README.md

```diff
@@ -229,3 +231,56 @@ StoreBuilder
 * `setExpireAfterTimeUnit(expireAfterTimeUnit: TimeUnit)` sets the time unit used when setting `expireAfterAccess` or `expireAfterWrite`. Default unit is `TimeUnit.SECONDS`.
 
 Note that `setExpireAfterAccess` and `setExpireAfterWrite` **cannot** both be set at the same time.
+
+### Clearing store entries
+
+You can delete a specific entry by key from a store, or clear all entries in a store.
+
+#### Store with no persister
+
+```kotlin
+val store = StoreBuilder
+  .fromNonFlow<Pair<String, RedditConfig, , List<Post>> {
```

Comment:
the example doesn't need to be this complicated. just use `fromNonFlow<String, Int>`